docs/scripts_md/Notify.md

```diff
@@ -0,0 +1,113 @@
+# NAME
+
+NeuroDB::Notify -- Provides an interface to the email notification subsystem
+of LORIS
+
+# SYNOPSIS
+
+    use NeuroDB::Notify;
+
+    my $notifier = NeuroDB::Notify->new(\$dbh);
+
+    my $message           = "Some kind of message from tarchive validation";
+    my $upload_id         = 123456;
+    my $notify_notsummary = 'N';
+    $notifier->spool('tarchive validation', $message,   0,
+                             'tarchiveLoader',      $upload_id, 'Y',
+                             $notify_notsummary
+                            );
```

Comment:
@cmadjar 
another small comment; the indentation does not look right here nor at the terminal when you type perldoc...

Reply:
weird. I have to disalign it in the code so that it looks aligned in perldoc... go figure...